admiral/pkg/controller/common/common.go

```diff
@@ -98,3 +95,12 @@ func GetNodeLocality(node *k8sV1.Node) string {
 	region, _ := node.Labels[NodeRegionLabel]
 	return region
 }
+
+func GetValueForKeyFromDeployment(key string, deployment *k8sAppsV1.Deployment) string {
```

Comment:
Can you check if we need to use this in deployment.go?

Reply:
This specifically isn't, but the 'identity' label was hardcoded in a number of places. Removed those.